### ISSUE TITLE
'add_autoload_singleton' wont work as expected

### ISSUE THREAD

**Issue by Gnumaru** [ellogwen/mesh-creator#1]
In editor.gd, at _enter_tree, you're using 'add_autoload_singleton' in order to register singletons, and then you try to access those singletons by name, as one would expect to work.

But unfortunately it doesn't work. You need to use Engine.get_singleton in order to retrieve the singletons, otherwise you get parse errors on godot 3.4.2. Maybe it would work on older versions, but on the latest stable version (3.4.2) it just doesn't work.

I had to replace every named singleton call with Engine.get_singleton in order to activate the plugin

**Comment by ellogwen** [comment]
Yeah, that feature is broken for plugins. Maybe it will get better with Godot 4 /GDScript 2
You can use this plugin https://github.com/jarneson/godot-ply
